src/hooks/useAccountEmptyState.js

```diff
@@ -11,8 +11,9 @@ export default function useAccountEmptyState() {
   useEffect(() => {
     const checkStorage = async () => {
       const reallyEmpty = await getAccountEmptyState(accountAddress, network);
-      if (reallyEmpty) {
-        setIsReallyEmpty(true);
+
+      if (!isEmpty && reallyEmpty) {
```

Comment:
logically, this results in the same true/false outcomes as the previous code

`setIsReallyEmpty` will always set `true` on line 16 (since `reallyEmpty` needs to be `true` on line 15)
and the one difference will be when `isEmpty` is `true`, which will result in `setIsReallyEmpty` on line 18 to set `true` anyway.

Can you point me to where the fix for the AddFunds interstitial being shown on fully empty wallets is?